pyproject.toml

```diff
@@ -27,6 +27,9 @@ dependencies = [
 where = ["."]
 exclude = ["tests"]
 
+[tool.setuptools.package-data]
```

Comment:
Maybe for a future pr, but I wonder if pyboy_states should be moved into the package and package data. Would that be useful for you?

Reply:
I see people are playing with pyboy_states a lot, so I think it's kind of early to move into package. We can revisit this when we have a baseline that solves pokemon.